enterprise/api/v1/urls.py

```diff
@@ -7,16 +7,15 @@
 
 from enterprise.api.v1 import views
 
-# Pylint considers module level variable as "Constant" and expects them to be upper-cased
-# that is why we have disabled 'invalid-name' check for variable definition below.
 router = DefaultRouter()  # pylint: disable=invalid-name
 router.register("site", views.SiteViewSet, 'site')
 router.register("auth-user", views.UserViewSet, 'auth-user')
-router.register(
-    "enterprise-course-enrollment",
-    views.EnterpriseCourseEnrollmentViewSet,
-    'enterprise-course-enrollment'
-)
+router.register("enterprise-customer-catalog", views.EnterpriseCustomerCatalogViewSet, 'enterprise-customer-catalog')
+router.register("enterprise-catalogs", views.EnterpriseCustomerCatalogApiViewSet, 'enterprise-catalogs')
```

Comment:
@UmanShahzad Why did we add two separate routes/ViewSets here?

`/enterprise/api/v1/enterprise-catalogs` should return the list of EnterpriseCatalog's that the requesting user has access to. As currently implemented, it returns the results of an unfiltered query to the catalog's search endpoint.